Sources/Overload/OvEditor/src/OvEditor/Panels/SceneView.cpp

```diff
@@ -204,7 +204,8 @@ void OvEditor::Panels::SceneView::HandleActorPicking()
 
 		m_actorPickingFramebuffer.Bind();
 		uint8_t pixel[3];
-		glReadPixels(static_cast<int>(mouseX), static_cast<int>(mouseY), 1, 1, GL_RGB, GL_UNSIGNED_BYTE, pixel);
+		auto& baseRenderer = *EDITOR_CONTEXT(renderer).get();
+		baseRenderer.ReadPixels(static_cast<int>(mouseX), static_cast<int>(mouseY), 1, 1, OvRendering::Settings::EPixelDataFormat::RGB,OvRendering::Settings::EPixelDataType::UNSIGNED_BYTE,pixel);
```

Comment:
Same here, I would suggest:

```suggestion
		EDITOR_CONTEXT(renderer)->ReadPixels(static_cast<int>(mouseX), static_cast<int>(mouseY), 1, 1, OvRendering::Settings::EPixelDataFormat::RGB, OvRendering::Settings::EPixelDataType::UNSIGNED_BYTE, pixel);
```